obsah/__init__.py

```diff
@@ -398,6 +398,10 @@ def generate_ansible_args(inventory_path, args, obsah_arguments):
         ansible_args.extend(["-e", extra_var])
 
     variables = {arg: getattr(args, arg) for arg in obsah_arguments if hasattr(args, arg)}
+
+    # Automatically expose obsah_state_dir variable to all playbooks
+    variables['obsah_state_dir'] = args.playbook.application_config.state_path()
```

Comment:
given the attribute is called `state_path`, we probably should keep the naming, so
```suggestion
    variables['obsah_state_path'] = args.playbook.application_config.state_path()
```